projects/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ add_subdirectory(22_image_filter)
 add_subdirectory(23_async_compute)
 add_subdirectory(27_mipmap_demo)
```

Comment:
I don't know if projects 24-26 are reserved for something, but perhaps makes sense to fill existing missing numbers?

Reply:
I don't think so. I've changed the project to use 24.